debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/SequenceObjectUpdater.java

```diff
@@ -62,6 +60,27 @@ public void initColumnSequenceMap(String columnSequenceMapString){
             this.sequenceMax.put(sequenceName, this.sequenceMax.getOrDefault(sequenceName, (long)0));
         }
     }
+    public void initSequenceMax(String initSequenceMapString, ConcurrentMap<String, Long> sequenceMax){
+        this.sequenceMax = new ConcurrentHashMap<>();
```

Comment:
A comment showing sample contents of the `initSequenceMapString` would really help here.